### ISSUE TITLE
The name property of the Rating control contains Unicode characters

### ISSUE THREAD

**Issue by naresh1021** [microsoft/fluentui#16169]
### Environment Information

- **Package version(s)**: 7.150.0
- **Browser and OS versions**: IE and Windows 10 ( 10.0.19042 )

### Describe the issue:

Using the Accessibility Insights for Windows on the Rating control shows the name property having some Unicode characters.

### Please provide a reproduction of the issue in a codepen: 

CodePen doesn't support IE. So not sure how to present this. You can open the [Fluent UI Rating page](https://developer.microsoft.com/en-us/fluentui#/controls/web/rating) on IE and reproduce this issue.

1. Open the page -> https://developer.microsoft.com/en-us/fluentui#/controls/web/rating
2. Open Accessibility Insights for Windows
3. Hover over the Rating control
3.1. Name property contains Unicode characters at the end
3.2 It is a violation of the Rule - "The Name property must not contain any characters in the private Unicode range.". Rule Source - [Section 508 502.3.1](https://www.access-board.gov/ict/)

This issue doesn't repro on other browsers namely Edge and Chrome. But reproducible only on Internet Explorer. 
[IWebBrowser2 control](https://docs.microsoft.com/en-us/windows/win32/api/exdisp/nn-exdisp-iwebbrowser2) uses Internet Explorer behind the scenes to render the web page as native control on the Win32 application. Hence this issue needs to be fixed as it becomes an Accessibility violation for the native app which hosts this web page using IWebBrowser2 control.

#### Actual behavior:

Name property shouldn't contain Unicode characters.

#### Expected behavior:

A couple of Unicode characters are present at the end of the Name property

### Documentation describing expected behavior

The object role, state(s), properties, boundary, name, and description shall be programmatically determinable.

https://www.access-board.gov/ict/#502.3.1

**Comment by ecraig12345** [comment]
I tried this and while Accessibility Insights (only the Windows version, not the browser extension for Edge or Chrome) flags this as a violation, I think it's a false positive. The elements with unicode characters are the icons for the stars, but as you can see in the HTML below, those elements (`<i>`) are marked as `aria-hidden="true"` and therefore shouldn't be read. I also confirmed with Narrator+IE that the icons aren't read.

Since this appears to be a bug with Accessibility Insights, I'm going to close this issue and would recommend filing a bug with the tool instead, [as described here](https://accessibilityinsights.io/docs/en/windows/getstarted/bugfiling/). 

```html
<button
  class="ms-Rating-button ratingButton-190 ms-Rating--large ratingStarIsLarge-192"
  id="Rating147-star-1"
  role="presentation"
  type="button"
  tabindex="-1"
>
  <span id="RatingLabel148-2" class="ms-Rating-labelText labelText-193"
    >Select 2 of 5 stars</span
  >
  <div class="ms-RatingStar-container ratingStar-187">
    <i
      data-icon-name="FavoriteStar"
      aria-hidden="true"
      class="ms-RatingStar-back ratingStarBack-195"
      ></i
    ><i
      data-icon-name="FavoriteStar"
      aria-hidden="true"
      class="ms-RatingStar-front ratingStarFront-196"
      style="width: 0%"
      ></i
    >
  </div>
</button>
```
